templates/kustomizeversions.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
```

Comment:
Should have a follow up issue to move conformance stuff moved into a separate directory so it doesn't confuse consumers.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1610